linkerd/app/core/src/control.rs

```diff
@@ -19,8 +34,54 @@ impl fmt::Display for ControlAddr {
     }
 }
 
+pub type Client<B> = Buffer<
+    http::Request<B>,
+    http::Response<
+        linkerd2_http_metrics::requests::ResponseBody<
+            http::balance::PendingUntilFirstDataBody<
+                tower::load::peak_ewma::Handle,
+                http::glue::Body,
+            >,
+            classify::Eos,
+        >,
+    >,
```

Comment:
Nit: Is it worth importing `http::balance`, `tower::load` and `http::glue` to potentially make this formatting less weird ?